var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -30,10 +30,13 @@ class Sz(AutotoolsPackage):
     """Error-bounded Lossy Compressor for HPC Data."""
 
     homepage = "https://collab.cels.anl.gov/display/ESR/SZ"
-    url      = "https://github.com/disheng222/SZ/archive/v1.4.11.0.tar.gz"
+    url      = "https://github.com/disheng222/SZ/archive/v1.4.13.4.zip"
```

Comment:
If you change the extension, none of the older versions will download. Can you keep it `.tar.gz` and update the checksums?